src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -257,11 +258,69 @@ private void AddComponentStats(StringBuilder builder)
                     + " agent:" + agent);
             }
 
+            var oneTryBuilder = new StringBuilder();
+            var whiteListedBuilder = new StringBuilder();
+            var addNodeBuilder = new StringBuilder();
+            var otherBuilder = new StringBuilder();
+            var addNodeDict = this.ConnectionSettings.AddNode.ToDictionary(ep => ep.MapToIpv6(), ep => ep);
+
+            foreach (INetworkPeer peer in this.ConnectedPeers)
+            {
+                bool added = false;
+
+                var connectionManagerBehavior = peer.Behavior<ConnectionManagerBehavior>();
+                if (connectionManagerBehavior.OneTry)
+                {
+                    AddPeerInfo(oneTryBuilder, peer);
+                    added = true;
+                }
+
+                if (connectionManagerBehavior.Whitelisted)
+                {
+                    AddPeerInfo(whiteListedBuilder, peer);
+                    added = true;
```

Comment:
A whitelisted node can also be an addnode so there will be a duplicate (if that is an issue or not I am not sure)

Reply:
I would go for simplicity here and perhaps accept that a node may be listed twice.

Reply:
I can live with that